runtime/onert/backend/train/ops/ElementwiseActivationLayer.cc

```diff
@@ -54,22 +55,31 @@ void ElementwiseActivationLayer::configure(const IPortableTensor *input, IPortab
     case ElementwiseActivationType::kReLU:
       if (input->data_type() == OperandType::FLOAT32)
       {
-        if (alpha == std::numeric_limits<float>::infinity() && beta == 0.f)
+        if ((alpha == std::numeric_limits<float>::infinity() || alpha == 6.0f) && beta == 0.f)
         {
           cpu::ops::ElementwiseActivationLayer::configure(
             input, output, alpha, beta, cpu::ops::ElementwiseActivationType::kReLU);
 
-          _backward_kernel = [](const IPortableTensor *output, const IPortableTensor *incoming,
-                                IPortableTensor *outgoing) {
-            nnfw::cker::train::ReLUGrad(getShape(output), getBuffer<float>(output),
-                                        getShape(incoming), getBuffer<float>(incoming),
-                                        getShape(outgoing), getBuffer<float>(outgoing));
+          auto relu_cker = [&alpha]() {
```

Comment:
`alpha` is a local variable. Please use capture by value.
```suggestion
          auto relu_cker = [alpha]() {
```

Reply:
I thought reference is fine,
Because this lambda function is invoked right now - when it is defined. 
This lambda function is also used in only local!
Is there anything I miss-understood here?

Anyway, alpha is just float type, removing & is no problem with me.

Reply:
Ah, I thought `relu_cker is a lambda, but it's not.

Reply:
Ah, definitely.. this using is tricky. :sweat_smile: